synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -146,45 +146,52 @@ studio::Dock_Info::Dock_Info()
 	g.set_tooltip_text(_("Green component value of color\nThe value after gamma correction, if different, is given in brackets"));
 	b.set_tooltip_text(_("Blue component value of color\nThe value after gamma correction, if different, is given in brackets"));
 	a.set_tooltip_text(_("Alpha component value of color, i.e. opacity"));
-	table->attach_next_to(r, *r_label, Gtk::POS_RIGHT, 1,1);
-	table->attach_next_to(g, *g_label, Gtk::POS_RIGHT, 1,1);
-	table->attach_next_to(b, *b_label, Gtk::POS_RIGHT, 1,1);
-	table->attach_next_to(a, *a_label, Gtk::POS_RIGHT, 1,1);
+	grid->attach_next_to(r, *r_label, Gtk::POS_RIGHT, 1, 1);
+	grid->attach_next_to(g, *g_label, Gtk::POS_RIGHT, 1, 1);
+	grid->attach_next_to(b, *b_label, Gtk::POS_RIGHT, 1, 1);
+	grid->attach_next_to(a, *a_label, Gtk::POS_RIGHT, 1, 1);
 
 	Gtk::Label *separator2 = manage(new Gtk::Label(" "));
-	table->attach(*separator2, 0,4,8,1);
+	grid->attach(*separator2, 0, 4, 8, 1);
 
-	//Render Progress Bar
+	// Render Progress Bar
+	Gtk::HBox *render_box = manage(new Gtk::HBox());
 	Gtk::Label *render_progress_label = manage(new Gtk::Label());
-	render_progress_label->set_markup(etl::strprintf("<b>%s</b>", _("Render Progress: ")));
-	table->attach_next_to(*render_progress_label, *separator2, Gtk::POS_BOTTOM, 8,1);
+	Gtk::Label *left_spacer_label = manage(new Gtk::Label());
```

Comment:
So this label caused the start margin to be bigger than the end margin

![screenshot](https://user-images.githubusercontent.com/24214509/112615921-1d48d000-8e49-11eb-926c-bfb4f3efe13c.png)

Reply:
I've added `left_spacer_label` as a space on the left side of the progress bar in order to center it vertically as shown here:
|<--left_space-->|progress-bar|<--right_space-->|
but maybe there's a better method? Didn't find any horizontal spacers or space holders in gtkmm.

Reply:
`set_margin_start(30)`?

Reply:
And same for stop button left margin that way widgets look properly center aligned.